### ISSUE TITLE
Error during install - spl-token-cli

### ISSUE THREAD

**Issue by fusion525** [solana-labs/solana-program-library#5762]
Hi,

I am getting the following error during the installation of spl-token-cli with cargo:

```
Compiling solana-account-decoder v1.16.3
error[E0432]: unresolved import `index_list::Index`
 --> /home/szabolcs/.cargo/registry/src/index.crates.io-6f17d22bba15001f/solana-runtime-1.16.3/src/read_only_accounts_cache.rs:5:18
  |
5 |     index_list::{Index, IndexList},
  |                  ^^^^^ no `Index` in the root
  |
  = help: consider importing one of these items instead:
          std::ops::Index
          core::ops::Index

   Compiling solana-transaction-status v1.16.3
For more information about this error, try `rustc --explain E0432`.
error: could not compile `solana-runtime` (lib) due to previous error
warning: build failed, waiting for other jobs to finish...
error: failed to compile `spl-token-cli v3.1.1`, intermediate artifacts can be found at `/tmp/cargo-installq8mOit`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.
```
**Versions:**
spl-token-cli version: `3.2.0 `
rustc version: `rustc 1.73.0 (cc66ad468 2023-10-03)`

**Previously I tried:**
`sudo apt-get install libudev-dev`
`sudo apt install pkg-config`
`cargo update`
`rustup update`

**Comment by buffalojoec** [comment]
Hi we're looking into this! #5737

**Comment by joncinque** [comment]
This has been fixed with version 0.2.11 of index-list https://crates.io/crates/index_list/0.2.11 -- resolving
